### ISSUE TITLE
Consider using either TLS endpoints or local endpoints but not both

### ISSUE THREAD

**Issue by mbyczkowski** [superfly/wormhole#47]
Initially, TLS endpoints started as a secure option for local endpoints (so fully encrypted connection even between local proxy - like nginx) that can be exposed (because of encryption and shared, static port). It seems unnecessarily complex to have wormhole listen on both TLS and dynamic ports opened for each session (so right now always 2 endpoints for each session).

@doodles526 I think we should make it either/or. I know we talked about having a support for both initially, but that constraint is gone. We could just make `UseSharedPortForwarding` turn off local endpoints. Thoughts?

**Comment by doodles526** [comment]
Yeah, I agree on that front. Using both at the same time was put there by me mostly as a way to make sure you all could easily transitiong existing connections. If that change is finished on your end I think having one or the ether is a solid decision
